src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -447,18 +449,44 @@ var DynamicTable = new Class({
 
     setSortedColumn: function(column) {
         if (column != this.sortedColumn) {
+            var oldColumn = this.sortedColumn;
             this.sortedColumn = column;
             this.reverseSort = '0';
+            this.setSortedColumnIcon(column, oldColumn, false);
         }
         else {
             // Toggle sort order
             this.reverseSort = this.reverseSort == '0' ? '1' : '0';
+            this.setSortedColumnIcon(column, null, (this.reverseSort == '1'));
         }
         localStorage.setItem('sorted_column_' + this.dynamicTableDivId, column);
         localStorage.setItem('reverse_sort_' + this.dynamicTableDivId, this.reverseSort);
         this.updateTable(false);
     },
 
+    setSortedColumnIcon: function(newColumn, oldColumn, isReverse) {
```

Comment:
this function is used only in `setSortedColumn: function(column) {}`, how about move it in there?

Reply:
It's also used in `setup()`